packages/FileUpload/src/FileList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import DocumentIcon from "../DocumentIcon";
+
+const IMAGE = "image";
+
+const FileList = ({
+  previewImages,
+  handleCancel
+}: {
+  previewImages: any;
+  handleCancel: any;
+}) => (
+  <>
+    {previewImages.map((file: any) => (
+      <React.Fragment key={file.id}>
```

Comment:
As I commented above, fix this to be consistent. You can use either `<>` or `Fragment` but don't mix them.